tests/inference/pipeline/test_preprocessing.py

```
@@ -128,18 +128,63 @@ def test_resolve_openvino_generic_npu_maps_to_concrete_id(self, prep_manager):
         prep_manager._device_id = "NPU"
         available = ["OpenVINOExecutionProvider"]
-
-        mock_core = mock.MagicMock()
-        mock_core.available_devices = ["CPU", "NPU.0"]
-
-        with (
-            mock.patch("openvino.Core", return_value=mock_core),
-            mock.patch("modules.inference.pipeline.preprocessing.config") as mock_cfg,
-        ):
+        with mock.patch("modules.inference.pipeline.preprocessing.config") as mock_cfg:
             mock_cfg.OV_CACHE_DIR = "/tmp/ov"
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/ov"

(S108)